src/block-components/typography/edit.js

```diff
@@ -0,0 +1,265 @@
+/**
+ * External dependencies
+ */
+import { unescape } from 'lodash'
+import { i18n } from 'stackable'
+import {
+	useAttributeEditHandlers, useBlockHoverState,
+} from '~stackable/hooks'
+import {
+	AdvancedRangeControl,
+	AdvancedSelectControl,
+	AdvancedToolbarControl,
+	AlignButtonsControl,
+	ButtonIconPopoverControl,
+	ColorPaletteControl,
+	FontFamilyControl,
+	HeadingButtonsControl,
+	InspectorStyleControls,
+	PanelAdvancedSettings,
+} from '~stackable/components'
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Fragment,
+} from '@wordpress/element'
+import { __, sprintf } from '@wordpress/i18n'
+import {
+	TextControl,
+} from '@wordpress/components'
+import { escapeHTML } from '@wordpress/escape-html'
+import { getAttributeName } from '~stackable/util'
+
+export const Edit = props => {
+	const {
+		label,
+		enableTextTag,
+		enableTextContent,
+		attrNameTemplate,
+		disableAlign,
+		withToggle,
+	} = props
+
+	const {
+		getAttribute,
+		updateAttribute,
+		updateAttributeHandler,
+		updateAttributes,
+	} = useAttributeEditHandlers( attrNameTemplate )
+
+	const [ state ] = useBlockHoverState()
+
+	return (
+		<InspectorStyleControls>
+			<PanelAdvancedSettings
+				title={ __( 'Text', i18n ) }
+				initialOpen={ true }
+				id="text"
+				{ ...{
+					...( withToggle ? {
+						hasToggle: withToggle,
+						checked: getAttribute( 'showText' ),
+						onChange: updateAttributeHandler( 'showText' ),
+					} : {} ),
+				} }
+			>
+				<Fragment>
+					{ enableTextContent && (
+						<TextControl
```

Comment:
I changed this to TextControl, but maybe we should have a prop `isMultiline: false` that you can use to make the content control into a TextareaControl